src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+            'using the scale where each category is assigned a serial',
+            'number from zero in the order it appears.'
+        ].join(' ')
+    }
+}));
+
+module.exports = {
+    mode: {
+        valType: 'flaglist',
+        editType: 'calc',
+        role: 'info',
+        flags: ['number', 'delta', 'gauge'],
+        dflt: 'number'
+    },
+    value: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        anim: true,
+        description: [
+            'Sets the number to be displayed.'
+        ].join(' ')
+    },
+    align: {
+        valType: 'enumerated',
+        values: ['left', 'center', 'right'],
+        role: 'info',
+        editType: 'plot',
+        description: [
+            'Sets the horizontal alignment of the `text` within the box.',
+            'Note that this attribute has no effect if an angular gauge is displayed:',
+            'in this case, it is always centered'
+        ].join(' ')
+    },
+    // position
+    domain: domainAttrs({name: 'indicator', trace: true, editType: 'calc'}),
+
+    vmin: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        dflt: 0,
+        description: [
+            'Sets the minimum value of the gauge.'
+        ].join(' ')
+    },
+    vmax: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        description: [
+            'Sets the maximum value of the gauge.'
+        ].join(' ')
+    },
+
+    title: {
+        text: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the title of this indicator.'
+            ].join(' ')
+        },
+        align: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the horizontal alignment of the title.',
+                'It defaults to `center` except for bullet charts',
+                'for which it defaults to right.'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display the title'
+            ].join(' ')
+        }),
+        editType: 'plot'
+    },
+    number: {
+        valueformat: {
+            valType: 'string',
+            dflt: '.3s',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the value formatting rule using d3 formatting mini-language',
+                'which is similar to those of Python. See',
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display main number'
+            ].join(' ')
+        }),
+        suffix: {
```

Comment:
I suspect will need a `prefix` attribute too, e.g for users wanting to show `$` before their formatted `value`.

Reply:
Added with tests in e6d8525ab07c56ca10bdbac6762b06d1d57bf830